src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -121,6 +121,10 @@ public override void Convert(Stream input, IResultLogWriter output, OptionallyEm
             _nodeDictionary.Clear();
             _snippetDictionary.Clear();
 
+            // Uncomment following Line to fetch FVDL content from any FPR file. This is not needed for conversion.
+            // However, we keep a copy of FVDL for each test file for debugging purposes.
+            // string fvdlContent = ExtractFvdl(OpenAuditFvdlReader(input));
```

Comment:
>// string fvdlContent = ExtractFvdl(OpenAuditFvdlReader(input)); [](start = 12, length = 64)

added this comment (and a simple method) for fvdl extraction from the fpr file. This is handy for new files.